src/demos/todomvc/todo-item.jsx

```diff
@@ -119,6 +121,15 @@ class TodoItem extends Component {
     return (
       <li className="m-todo__item">
         <div className="m-todo__wrap">
+          <AXADropDownReact
+            title="Switch status"
+            value={completed ? 'selected' : 'unselected'}
```

Comment:
I appreciate the `value` prop, I think we should follow react's principle here for all input components

Reply:
which is?

Reply:
i emulated all input principles.. all user-inputs HTML tags have value as attribute to set value, and this is what i emulated. What do you mean with "we should follow react's principle"? The sentence as it is unfortunately says nothing

Reply:
Sorry I meant to reference all [React's Form concepts](https://reactjs.org/docs/forms.html):
- [Controlled Components](https://reactjs.org/docs/forms.html#controlled-components)
- `[value=""]` attribute over HTML spec for [`<textarea>`](https://reactjs.org/docs/forms.html#the-textarea-tag), [`<select>`](https://reactjs.org/docs/forms.html#the-select-tag)

Reply:
Ahh ok.. well i think is quite similar how we started now isn't it?

Reply:
but anyway as best practice for future design desicions, we should keep to HTML standard/way to do and not react